spot-electron/src/application-menu/applicationmenu.js

```diff
@@ -21,4 +21,4 @@ const menuTemplate = [
 
 const menu = Menu.buildFromTemplate(menuTemplate);
 
-Menu.setApplicationMenu(menu);
+// Menu.setApplicationMenu(menu);
```

Comment:
What is this change for?

Reply:
That was a remainder of a debugging.... Remover, thanks